docs/VLSI/HAMMER.rst

```diff
@@ -1,10 +1,14 @@
+.. _hammer:
+
 Core HAMMER
```

Comment:
A general comment is that there is an inconsistency with using "Hammer" and "HAMMER". We should grep/sed and choose a single version of this throughout the docs.